code/src/Header.css

```diff
@@ -0,0 +1,37 @@
+.header-content{
```

Comment:
I like that you've separated out your css files, it makes it very easy to follow which bit it refers to.